<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название команды 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="command0Name"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Название команды 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="command1Name"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Включение/выключение раундов игры:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRound_1">
        <property name="text">
         <string>Простая игра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRound_2">
        <property name="text">
         <string>Двойная игра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRound_3">
        <property name="text">
         <string>Тройная игра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRound_Obr">
        <property name="text">
         <string>Игра наоборот</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRound_Super">
        <property name="text">
         <string>Супер игра</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
